src/main/resources/feedbackQuestionRubricEditFormTableOptionsFragment.html

```diff
@@ -0,0 +1,16 @@
+<td class="align-center rubricCol-${questionIndex}-${col}" data-col="${col}">
+    <div class="btn-group">
+        <button type="button" class="btn btn-default" id="rubricMoveChoiceLink-${questionIndex}-${col}-l" onclick="swapRubricCol(${col}, ${questionIndex}, true)"
+            data-toggle="tooltip" data-placement="top" title="Move column left">
```

Comment:
I didn't know that :sweat_smile: . Will do. 

Side note, can't we check something like this in the style checks?